src/components/FormValidator.js

```diff
@@ -57,7 +59,7 @@ export default class FormValidator {
     });
   }
   toggleButtonState() {
-    if (this._hasInvalidInput(this._inputElements)) {
+    if (this._hasInvalidInput()) {
       this._submitButtons.classList.add(this._inactiveButtonClass);
```

Comment:
`_submitButtons` should be without `s` because it's **one** button

`_submitButton`